lib/beamchmark/suite/measurements/cpu_info.ex

```diff
@@ -15,18 +15,15 @@ defmodule Beamchmark.Suite.Measurements.CpuInfo do
   use Bunch.Access
   alias Beamchmark.Math
 
-  @typedoc """
-  Single core snap statistics gathered via snapshot
-  """
-  @type cpu_core_usage_t ::
-          %{(core_id :: integer()) => usage :: Math.percent_t() | Math.percent_diff_t()}
-
   @typedoc """
   All information gathered via single snapshot + processor average
   """
   @type cpu_usage_t ::
```

Comment:
I am wondering if we shouldn't rename this to `cpu_snapshot_t()` :thinking: It would be more consistent with other typespecs and function names.